### DISCUSSION TITLE
ERROR: giscus.app not found

### DISCUSSION THREAD

**Discussion by tonymitchell** [giscus/giscus#797] [General]
The giscus comments section on my site weren't displaying and traced it to a 404 trying to fetch the client.js from giscus.app.  It appears the giscus.app is no longer being hosted/deployed.
![image](https://user-images.githubusercontent.com/1694080/203341844-d55f3190-ef93-4348-a008-9dc58c452807.png)

Should we be self-hosting this client.js file?

**Comment by laymonage** [comment]
Umm, it works fine here. Maybe just a temporary error?

**Reply by tonymitchell** [reply]
I'm still getting the error here on the west coast of Canada.  Perhaps this is a regional CDN issue with Vercel?  The link on the 404 page points to Vercel documentation. [giscus.app might be available. Click here to learn how to assign it to a project.](https://vercel.com/docs/custom-domains#assigning-a-domain-to-a-git-branch)

Every time I refresh I get a new ID on the 404 page, so it seems like the hosting provider is not finding the site (at least in my location).

**Reply by tonymitchell** [reply]
If it's helpful, here are the client.js request's response headers I get in Chrome dev tools when browsing my site:
![image](https://user-images.githubusercontent.com/1694080/203350314-e4e9e2fe-a4f5-483f-a014-85a62b8a47fb.png)

**Reply by laymonage** [reply]
Huh, that's possible. I've redeployed, does it work now?

https://geotargetly.com/geo-browse?url=https://giscus.app/client.js&country_code=CA shows up correctly here

**Reply by tonymitchell** [reply]
Yes, it's working now. Thanks!
